### PR TITLE
Improves Code Readability for Syndi grenades

### PR DESCRIPTION
**Pull request by YakumoChen**
## About The Pull Request
Gives vars better names
Gives me the GBP

## Why It's Good For The Game
Coding guidelines are cool

## Changelog
code improvement

### CODE REVIEW

File: code/game/objects/items/grenades/syndieminibomb.dm [hunk 1]
```diff
@@ -59,11 +59,11 @@
...
 	playsound(loc, 'sound/effects/empulse.ogg', 50, TRUE)
 	radiation_pulse(src, rad_damage)
-	for(var/turf/T in view(freeze_range,loc))
-		if(isfloorturf(T))
-			var/turf/open/floor/F = T
-			F.MakeSlippery(TURF_WET_PERMAFROST, 6 MINUTES)
-			for(var/mob/living/carbon/L in T)
-				L.adjustStaminaLoss(stamina_damage)
-				L.adjust_bodytemperature(-230)
+	for(var/turf/Turf in view(freeze_range,loc))
```
Comment (Mothblocks) [inline code/game/objects/items/grenades/syndieminibomb.dm:62]: This is not a descriptive name.

You need to describe the *context* the variable is in. Re-read https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Also, snake_case.

File: code/game/objects/items/grenades/syndieminibomb.dm [hunk 2]
```diff
@@ -59,11 +59,11 @@
...
+		if(isfloorturf(Turf))
+			var/turf/open/floor/ItsAFloor = Turf
```
Comment (Mothblocks) [inline code/game/objects/items/grenades/syndieminibomb.dm:64]: Same here.

File: code/game/objects/items/grenades/syndieminibomb.dm [hunk 3]
```diff
@@ -59,11 +59,11 @@
...
+			ItsAFloor.MakeSlippery(TURF_WET_PERMAFROST, 6 MINUTES)
+			for(var/mob/living/carbon/Victim in Turf)
```
Comment (Mothblocks) [inline code/game/objects/items/grenades/syndieminibomb.dm:66]: snake_case

**Review by Mothblocks — CHANGES_REQUESTED** [review]

**Comment by YakumoChen** [comment]
@Mothblocks is this better

File: code/game/objects/items/grenades/syndieminibomb.dm [hunk 4]
```diff
@@ -59,11 +59,11 @@
...
-	for(var/turf/T in view(freeze_range,loc))
-		if(isfloorturf(T))
-			var/turf/open/floor/F = T
-			F.MakeSlippery(TURF_WET_PERMAFROST, 6 MINUTES)
-			for(var/mob/living/carbon/L in T)
-				L.adjustStaminaLoss(stamina_damage)
-				L.adjust_bodytemperature(-230)
+	for(var/turf/turf_affected in view(freeze_range,loc))
+		if(isfloorturf(turf_affected))
+			var/turf/open/floor/its_floor = turf_affected
```
Comment (Mothblocks) [inline code/game/objects/items/grenades/syndieminibomb.dm:64]: just `floor`

**Comment by Mothblocks** [comment]
almost

File: code/game/objects/items/grenades/syndieminibomb.dm [hunk 5]
```diff
@@ -59,11 +59,11 @@
...
 	radiation_pulse(src, rad_damage)
-	for(var/turf/T in view(freeze_range,loc))
-		if(isfloorturf(T))
-			var/turf/open/floor/F = T
-			F.MakeSlippery(TURF_WET_PERMAFROST, 6 MINUTES)
-			for(var/mob/living/carbon/L in T)
-				L.adjustStaminaLoss(stamina_damage)
-				L.adjust_bodytemperature(-230)
+	for(var/turf/turf_affected in view(freeze_range,loc))
+		if(isfloorturf(turf_affected))
```
Comment (Mothblocks) [inline code/game/objects/items/grenades/syndieminibomb.dm:63]: Last thing, didn't notice this until now--sorry, chang this to `for (var/turf/open/floor/floor in view(freeze_range, loc))`.

If we're performing a type-check anyway, we might as well do it in the definition.

File: code/game/objects/items/grenades/syndieminibomb.dm [hunk 6]
```diff
@@ -59,7 +59,7 @@
 	update_mob()
 	playsound(loc, 'sound/effects/empulse.ogg', 50, TRUE)
 	radiation_pulse(src, rad_damage)
-	for(var/turf/turf_affected in view(freeze_range,loc))
+	for (var/turf/open/floor/floor in view(freeze_range, loc))
 		if(isfloorturf(turf_affected))
```
Comment (Mothblocks) [inline code/game/objects/items/grenades/syndieminibomb.dm:63]: You don't need this anymore.

**Comment by YakumoChen** [comment]
@Mothblocks I assume changing it to floor_affected wouldn't work
Code standards are just confusing now.

**Comment by YakumoChen** [comment]
I fucked up and am no longer sure how

**Comment by YakumoChen** [comment]
it turns out deleting more code was the answer

**Comment by Mothblocks** [comment]
usually is 😎

**Comment by Mothblocks** [comment]
Sorry for delays, all good.

**Pull request merged**